amostra/testing.py

```diff
@@ -1,55 +1,25 @@
-import pytest
+from pymongo import MongoClient
+import time as ttime
+from amostra.ignition import start_server
+import uuid
 
 
-class _baseSM:
```

Comment:
The idea here was to have a 'base' set of tests that we could use across multiple implementations of this API